### ISSUE TITLE
Rename "work orders" to "cases" throughout the application

### ISSUE THREAD

**Issue by lamara11** [amosproj/amos2024ss01-xcelerator-demo-app#76]
## User story

1. As a facility manager
2. I want to have the work orders renamed to cases 
3. So that the naming is compliant with Siemens' naming. 

## Acceptance criteria

- [x] Work orders are renamed to Cases throughout the application

## Definition of done (DoD)

- [x] Added only after week 5
- [x] The same for all features
- [x] Here goes the project specific part

## DoD general criteria

- [x] Feature has been fully implemented
- [x] Feature has been merged into the mainline
- [x] All acceptance criteria were met
- [x] Product owner approved features
- [x] All tests are passing
- [x] Developers agreed to release
